app/components/MovieCard/MovieCard.jsx

```diff
@@ -1,13 +1,18 @@
-import "./MovieCard.css";
 import { Link } from "@remix-run/react";
+import styles from "./MovieCard.css";
 
 export default function MovieCard(props) {
-  console.log(props);
   const movie = props;
```

Comment:
doesn't make sense.
Add desctructuring, convert file to tsx (if build tool is set up)